tests/hmm/hmumu_utils.py

```diff
@@ -577,10 +580,26 @@ def compute_event_weights(weights, scalars, genweight_scalefactor, pu_correction
             print("pu_weights_up", pu_weights_up.mean(), pu_weights_up.std())
             print("pu_weights_down", pu_weights_down.mean(), pu_weights_down.std())
 
-        weights["puWeight_off"] = weights["nominal"] 
-        weights["puWeight__up"] = weights["nominal"] * pu_weights_up
-        weights["puWeight__down"] = weights["nominal"] * pu_weights_down
-        weights["nominal"] = weights["nominal"] * pu_weights
+        if NUMPY_LIB.logical_or(dataset_era == "2016",dataset_era == "2017"):
```

Comment:
small clarification - logical_or is only needed if you are dealing with arrays, `dataset_era` is a simple string and you can use the usual python booleans (`or`, `and`) etc.

Reply:
Thanks a lot for the suggestion! Indeed I should use the use the usual python booleans in this case.